src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -140,7 +140,9 @@ public void run() {
 
                 // Save the database
                 success = saveDatabase(panel.getBibDatabaseContext().getDatabaseFile().get(), false,
-                        panel.getBibDatabaseContext().getMetaData().getEncoding()
+                        panel.getBibDatabaseContext()
```

Comment:
Do these restylings happen automatically? If yes, could eclipse be configured differently? Sometimes they are good, sometimes they feel a bit over aggressive (like here, the two-lines solution was fine in my opinion)